pkg/operator/datasync/validation.go

```diff
@@ -78,7 +78,20 @@ func validateClientCert(pem []byte) []error {
 	return errs
 }
 
-func validatePrivateKey(pemKey []byte) []error {
+func ValidateServerCert(pem []byte) []error {
+	errs := []error{}
+
+	certs, certErrs := parseCerts(pem)
+	errs = append(errs, certErrs...)
+
+	if numCerts := len(certs); numCerts != 1 {
+		return append(errs, fmt.Errorf("expected a single server certificate, got %d", numCerts))
+	}
```

Comment:
The server cert PEM will likely contain the full certificate chain so I would remove this check

Reply:
Will update the check to expect at least one cert.